x-pack/legacy/plugins/monitoring/server/lib/metrics/__test__/__snapshots__/metrics.test.js.snap

```diff
@@ -2,6 +2,416 @@
+      },
+      "event_rate": Object {
+        "sum_bucket": Object {
+          "buckets_path": "beats_uuids>event_rate_per_beat",
+          "gap_policy": "skip",
+        },
+      },
+      "metric_deriv": Object {
+        "derivative": Object {
+          "buckets_path": "event_rate",
+          "gap_policy": "skip",
+          "unit": "1m",
+        },
+      },
+    },
+    "app": "apm",
+    "derivative": true,
+    "description": "HTTP invalid query",
+    "field": "beats_stats.metrics.apm-server.acm.response.errors.invalidquery",
+    "format": "0,0.[00]",
+    "label": "Invalid Query",
+    "metricAgg": "max",
+    "timestampField": "beats_stats.timestamp",
+    "title": "Invalid Query",
+    "units": "/m",
+    "uuidField": "cluster_uuid",
+  },
+  "apm_acm_responses_errors_method": ApmEventsRateClusterMetric {
+    "aggs": Object {
+      "beats_uuids": Object {
+        "aggs": Object {
+          "event_rate_per_beat": Object {
+            "max": Object {
+              "field": "beats_stats.metrics.apm-server.acm.response.errors.method",
+            },
+          },
+        },
+        "terms": Object {
+          "field": "beats_stats.beat.uuid",
+          "size": 10000,
+        },
+      },
+      "event_rate": Object {
+        "sum_bucket": Object {
+          "buckets_path": "beats_uuids>event_rate_per_beat",
+          "gap_policy": "skip",
+        },
+      },
+      "metric_deriv": Object {
+        "derivative": Object {
+          "buckets_path": "event_rate",
+          "gap_policy": "skip",
+          "unit": "1m",
+        },
+      },
+    },
+    "app": "apm",
+    "derivative": true,
+    "description": "HTTP Requests rejected due to incorrect HTTP method",
+    "field": "beats_stats.metrics.apm-server.acm.response.errors.method",
+    "format": "0,0.[00]",
+    "label": "Method",
+    "metricAgg": "max",
+    "timestampField": "beats_stats.timestamp",
+    "title": "Method",
+    "units": "/m",
+    "uuidField": "cluster_uuid",
+  },
+  "apm_acm_responses_errors_unauthorized": ApmEventsRateClusterMetric {
+    "aggs": Object {
+      "beats_uuids": Object {
+        "aggs": Object {
+          "event_rate_per_beat": Object {
+            "max": Object {
+              "field": "beats_stats.metrics.apm-server.acm.response.errors.forbidden",
+            },
+          },
+        },
+        "terms": Object {
+          "field": "beats_stats.beat.uuid",
+          "size": 10000,
+        },
+      },
+      "event_rate": Object {
+        "sum_bucket": Object {
+          "buckets_path": "beats_uuids>event_rate_per_beat",
+          "gap_policy": "skip",
+        },
+      },
+      "metric_deriv": Object {
+        "derivative": Object {
+          "buckets_path": "event_rate",
+          "gap_policy": "skip",
+          "unit": "1m",
+        },
+      },
+    },
+    "app": "apm",
+    "derivative": true,
+    "description": "Forbidden HTTP Requests rejected - CORS violation, disabled enpoint",
```

Comment:
Please update description to something like `Unauthorized HTTP Requests rejected - invalid auth token`